docs/using-the-jdbc-driver/CompatibilityEndpoints.md

```diff
@@ -0,0 +1,125 @@
+| [driverMetaData](./using-plugins/UsingTheDriverMetadataConnectionPlugin.md)           | <span style="color:yellow;font-size:15px">&check;</span>                                                                                                   | <span style="color:yellow;font-size:15px">&check;</span>                                                                                                   | <span style="color:yellow;font-size:15px">&check;</span>                                                    | <span style="color:yellow;font-size:15px">&check;</span>                                                                        |
+| connectTime                                                                           | <span style="color:yellow;font-size:15px">&check;</span>                                                                                                   | <span style="color:yellow;font-size:15px">&check;</span>                                                                                                   | <span style="color:yellow;font-size:15px">&check;</span>                                                    | <span style="color:yellow;font-size:15px">&check;</span>                                                                        |
+| [dev](./using-plugins/UsingTheDeveloperPlugin.md)                                     | <span style="color:yellow;font-size:15px">&check;</span>                                                                                                   | <span style="color:yellow;font-size:15px">&check;</span>                                                                                                   | <span style="color:yellow;font-size:15px">&check;</span>                                                    | <span style="color:yellow;font-size:15px">&check;</span>                                                                        |
+| fastestResponseStrategy                                                               | <span style="color:yellow;font-size:15px">&check;</span>                                                                                                   | <span style="color:yellow;font-size:15px">&check;</span>                                                                                                   | <span style="color:yellow;font-size:15px">&check;</span>                                                    | <span style="color:red;font-size:20px">&cross;</span>                                                                           |
```

Comment:
Double checking, do these plugins actually work with rds proxy? I wasn't sure since I don't think we've done much testing with rds proxy
- failover 1/2
- auroraConnectionTracker
- fastestResponse
- bg